lnprototest/runner.py

```diff
@@ -12,14 +12,7 @@
 from .utils import privkey_expand
 from .keyset import KeySet
 from abc import ABC, abstractmethod
-from typing import (
-    Dict,
-    Optional,
-    List,
-    Union,
-    Any,
-    Callable,
-)
```

Comment:
There is any reason to remove this formatting here in favor of a list?

Reply:
the `black` formatter flagged it iirc!

Reply:
Ah ok! maybe there is some update on the black side :) it is always good someone that take care of the fmt imho